### ISSUE TITLE
`numberOfMonths` propType ignored in `DateRangePicker` and `SingleDatePicker` when orientation is set to `vertical`

### ISSUE THREAD

**Issue by byrondevonwall** [airbnb/react-dates#469]
When trying to set the `numberOfMonths` propType on either the single or range picker, the pickers will display 2 months no matter how many months are specified in the prop when the orientation is set to `vertical`.

The prop works as expected when orientation is set to `horizontal`.

**Comment by ljharb** [comment]
[This line](https://github.com/airbnb/react-dates/blob/master/src/components/DayPicker.jsx#L684) indeed explicitly only considers `numberOfMonths` when the orientation is not vertical.

Perhaps @majapw can weigh in on whether this is intentional; if so, then the propType should probably error when vertical and provided; if not, let's fix it.

**Comment by byrondevonwall** [comment]
@ljharb Yeah, it's not that big of a deal, I was just kind of banging my head against the wall trying to get it to work for some number of minutes before switching the orientation.  If it's intentional it would be nice if it were in the docs somewhere is all.  Thanks for the response!

**Comment by majapw** [comment]
In general, the `vertical` orientation is slightly less well-developed than the horizontal orientation so... this is something that should be fixed.
